app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/SuggestionFeedContent.kt

```diff
@@ -0,0 +1,235 @@
+    navigationActions: NavigationActions,
+    suggestionList: List<Suggestion>, // <-todo: will be real data (wait for William)
+    searchText: String
+) {
+
+  //    // Filter suggestions of a trip by userName based on search text todo: for sprint3
+  //    val filteredSuggestionsByTitle =
+  //        if (searchText.isEmpty()) {
+  //            suggestionList
+  //        } else {
+  //            suggestionList.filter { suggestion ->
+  // suggestion.userName.lowercase().contains(searchText.lowercase()) }
+  //        }
+
+  // Example usage of dummy data for the suggestionList <-todo: change for sprint3
+  val stop1 =
+      Stop(
+          stopId = "OSK001",
+          title = "Osaka Castle",
+          address = "1-1 Osakajo, Chuo Ward, Osaka, 540-0002, Japan",
+          date = LocalDate.of(2024, 4, 10), // Example date
+          startTime = LocalTime.of(9, 0), // Opens at 9 AM
+          duration = 120, // 2 hours visit
+          budget = 600.0, // Entrance fee and other expenses
+          description =
+              "Osaka Castle is one of Japan's most famous landmarks and it played a major role in the unification of Japan during the sixteenth century.",
+          geoCords = GeoCords(latitude = 34.687315, longitude = 135.526201),
+          website = "https://www.osakacastle.net/",
+          imageUrl = "")
+
+  val stop2 =
+      Stop(
+          stopId = "OSK002",
+          title = "Dotonbori",
+          address = "Dotonbori, Chuo Ward, Osaka, 542-0071, Japan",
+          date = LocalDate.of(2024, 4, 10),
+          startTime = LocalTime.of(18, 0), // Best experienced in the evening
+          duration = 180, // Approximately 3 hours
+          budget = 3000.0, // Food, shopping, and other activities
+          description =
+              "Dotonbori is Osaka's most famous tourist destination, known for its bright neon lights, extravagant signage, and abundant dining options.",
+          geoCords = GeoCords(latitude = 34.668723, longitude = 135.501295),
+          website = "https://www.dotonbori.or.jp/en/",
+          imageUrl = "")
+
+  val stop3 =
+      Stop(
+          stopId = "OSK003",
+          title = "Umeda Sky Building",
+          address = "1-1-88 Oyodonaka, Kita Ward, Osaka, 531-0076, Japan",
+          date = LocalDate.of(2024, 4, 11),
+          startTime = LocalTime.of(10, 30), // Opens at 10:30 AM
+          duration = 90, // 1.5 hours visit
+          budget = 1500.0, // Entrance fee and other possible expenses
+          description =
+              "The Umeda Sky Building is a spectacular high rise building in the Kita district of Osaka, featuring a futuristic observatory, the Floating Garden.",
+          geoCords = GeoCords(latitude = 34.705938, longitude = 135.490357),
+          website = "http://www.kuchu-teien.com/",
+          imageUrl = "")
+
+//  /*
+      //these data are used for checking the UI, will be replaced with real data with William
+      val suggestionList = listOf(

```

Comment:
careful with variable with same name, had trouble getting it to work locally

Reply:
Changed the variable name to `_suggestionList`, it's been working locally